packages/host/tests/integration/realm-indexing-and-querying-test.gts

```diff
@@ -3401,6 +3401,12 @@ module(`Integration | realm indexing and querying`, function (hooks) {
                 { id: mangoID },
                 {
                   id: vanGoghID,
+                  firstName: 'Van Gogh',
+                  friends: [
+                    {
+                      id: hassanID,
+                    },
+                  ],
```

Comment:
this is an interesting development--this search doc actually looks more correct now